rebuild/_config.scss

```diff
@@ -1,3 +1,4 @@
+$brand-assets-dir: '../assets/' !default;
```

Comment:
May need to remove the trailing slash here, or remove the first slash `rebuild/config/_icons.scss` at `'/images/...`.

If this is compiled as-is, the CSS will have a double slash e.g.

```
background-image:url(../assets//images/icons/hamburger-close--primary.svg)
```

Reply:
Have removed slash from paths in _icons